packages/abstraxion/src/index.ts

```diff
@@ -1,6 +1,7 @@
 import "./styles.css";
 
 export { Abstraxion, AbstraxionProvider } from "./components/Abstraxion";
+export { SignArbSecp256k1HdWallet } from "./SignArbSecp256k1HdWallet";
```

Comment:
Can you expose this initiated to the end user with the deserialized key (if available)?